test/yul/bitwise-coverage/Bitwise.js

```diff
@@ -0,0 +1,80 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const { expect } = require('chai')
+const { ethers } = require('hardhat')
+
+describe('@solidityequiv5 Bitwise Tests', () => {
```

Comment:
how are we deciding what number to put after the `solidityequiv`?

Reply:
Ah for this I thought with this new batch of test suites for Yul support I just give it a new number. I believe before the Yul support, it went up to `solidityequiv4` so I just made it `solidityequiv5`